.coveragerc

```diff
@@ -0,0 +1,8 @@
+[run]
+branch=True
+source=minion
```

Comment:
Do we really need the `minion` app? Is that the real coverage source for this project?